packages/web/src/pages/audio-rewards-page/components/modals/ChallengeRewardsModal/ChallengeRewardsModal.tsx

```diff
@@ -555,10 +593,21 @@ const ChallengeRewardsBody = ({ dismissModal }: BodyProps) => {
                   amount={formatNumberCommas(challenge?.totalAmount ?? '')}
                   subtext={messages.audio}
                 />
-              </div>
-              {renderProgressBar()}
-              {renderProgressStatusLabel()}
-            </div>
+              </Flex>
+              <Flex
```

Comment:
maybe also paper? not sure of context on this one though

Reply:
paper here gives us the rounded borders. possible to override but i think flex is ok?